pkg/repository/provider/unified_repo.go

```diff
@@ -586,6 +586,11 @@ func getStorageVariables(backupLocation *velerov1api.BackupStorageLocation, repo
 		validParams := []string{
 			udmrepo.StoreOptionCacheLimit,
 			udmrepo.StoreOptionKeyFullMaintenanceInterval,
+			udmrepo.ThrottleOptionReadOps,
```

Comment:
At present, we only have the internal processing for the throttling logics, the options are not exposed.
Since we are exposing them, let's give them better names to show they are for throttling, e.g.,
`ThrottleUploadBytes`
`ThrottleDownloadBytes`
`ThrottleReadOPS`
`ThrottleWriteOPS`
`ThrottleListOPS`

Reply:
Sure, updated